### DISCUSSION TITLE
RestorePhotos照片修復 去模糊

### DISCUSSION THREAD

**Discussion by pierre0210** [1101-GS4719-C/Python-Community-GS4719-C#1099] [優秀專案]
## 簡介
[RestorePhotos](https://github.com/Nutlope/restorePhotos)是一款用Next.js框架寫成的工具，其AI model是採用前陣子很熱門的[GFPGAN](https://github.com/TencentARC/GFPGAN)，經過網頁包裝後更加user friendly。
根據他在[推特](https://twitter.com/nutlope/status/1612488923716136962)上發表的使用結果可以發現透過簡單的檔案拖拉就可以將畫質較低的照片經過處理轉換成更清晰的版本。

![](https://github.com/Nutlope/restorePhotos/raw/main/public/screenshot.png)

**Comment by Valentinaa0214** [comment]
這個技術太棒了吧，看來許多模糊的舊照片或是手抖拍出來的模糊照能夠經由這個技術加以修復了！
使用者友善的介面讓我也很想要拿幾張舊照片來試試看

**Comment by s96015welson** [comment]
隨著AI快速進步，影像修復技術也愈來愈強大了，期望這技術能在文物修復上做出貢獻，保存更多的回憶，或能使一些破舊無法辨識的古物恢復原貌。

**Comment by HsuChen88** [comment]
這項技術非常適合我，擁有最好的相機卻有無法控制的手抖，各位有救了。

**Comment by BlahFish1006** [comment]
這個技術好實用哦！不僅可以成為手抖怪的防抖救星，還能修復那些爺爺奶奶珍藏的老照片，
若能將這項技術應用在考古及文化修復的工作上，感覺會更加如虎添翼，
能幫助釐清舊時照片、圖畫、舊文件等等遺跡中的蛛絲馬跡。
